ui-particles-angular/projects/ui-particles-angular/src/lib/gio-form-json-schema/testing-json-schema/endpoints.ts

```diff
@@ -38,7 +38,7 @@ export const endpointsGetResponse = [
.password","description":"The store password for the key store file. This is optional for client and only needed if ssl.keystore.location is configured. Key store password is not supported for PEM format","type":"string"},"certificateChain":{"title":"ssl.keystore.certificate.chain","description":"Certificate chain in the format specified by \'ssl.keystore.type\'. Default SSL engine factory supports only PEM format with a list of X.509 certificates.","type":"string"},"key":{"title":"ssl.keystore.key","description":"Private key in the format specified by \'ssl.keystore.type\'. Default SSL engine factory supports only PEM format with PKCS#8 keys. If the key is encrypted, key password must be specified using \'ssl.key.password\'.","type":"string"},"keyPassword":{"title":"ssl.key.password","description":"The password of the private key in the key store file. This is optional for client.","type":"string"}},"allOf":[{},{"oneOf":[{"required":["location"]},{"required":["key","certificateChain"]}]},{"oneOf":[{"properties":{"type":{"enum":["JKS"]}},"not":{"required":["certificateChain"]}},{"properties":{"type":{"enum":["PKCS12"]}},"not":{"required":["certificateChain"]}},{"properties":{"type":{"enum":["PEM"]}},"not":{"required":["password"]}}]}],"required":["type"]}},"required":["trustStore"]}},"oneOf":[{"properties":{"protocol":{"enum":["PLAINTEXT"]}}},{"properties":{"protocol":{"enum":["SASL_PLAINTEXT"]}},"required":["sasl"]},{"properties":{"protocol":{"enum":["SASL_SSL"]}},"required":["sasl"]},{"properties":{"protocol":{"enum":["SSL"]}},"required":["ssl"]}]},"consumer":{"type":"object","title":"Consumer configuration","id":"urn:jsonschema:io:gravitee:plugin:endpoint:kafka:configuration:KafkaEndpointConnectorConfiguration:Consumer","properties":{"enabled":{"title":"enabled","description":"Allow to enable or disable the consumer capability.","type":"boolean","default":true},"encodeMessageId":{"title":"encodeMessageId","description":"This options allows encoding message id in base64.","type":"boolean","default":true},"autoOffsetReset":{"title":"auto.offset.reset","description":"What to do when there is no initial offset in Kafka or if the current offset does not exist any more on the server (e.g. because that data has been deleted): <ul><li>earliest: automatically reset the offset to the earliest offset<li>latest: automatically reset the offset to the latest offset</li><li>none: throw exception to the consumer if no previous offset is found for the consumer\'s group</li><li>anything else: throw exception to the consumer.</li></ul>","type":"string","default":"latest","enum":["latest","earliest","none"]}}},"producer":{"type":"object","title":"Producer configuration","id":"urn:jsonschema:io:gravitee:plugin:endpoint:kafka:configuration:KafkaEndpointConnectorConfiguration:Producer","properties":{"enabled":{"title":"enabled","description":"Allow to enable or disable the producer capability.","type":"boolean","default":"true"}}}},"required":["bootstrapServers"]}',
```

Comment:
I modified this value manually by editing the json and unformat it, so line breaks have been removed. But maybe there is a better way to do it...

Reply:
Maybe we can put this in json file

Reply:
i think we can update all plugins and the get again this config and update this lines (first comment of this file)
https://github.com/gravitee-io/gravitee-ui-particles/pull/199/files#diff-95b5b4ad47a25a6b8a1e7d14dd524273794f2831f3c952e4ac290c454c91ecdeR16-R19

Reply:
maybe revet this change for now 🤷‍♂️ as you wish , because the initial goal more to show the real listener config not the fixture